internal/metarepos/raft_metadata_repository.go

```diff
@@ -991,7 +991,12 @@ func (mr *RaftMetadataRepository) applyUnseal(r *mrpb.Unseal, nodeIndex, request
 		return err
 	}
 
-	mr.reportCollector.Unseal(r.LogStreamID, mr.GetLastCommitVersion())
+	ver := mr.GetLastCommitVersion()
+	if ver > 0 {
+		// Let logStream know cur version.
+		ver = ver - 1
+	}
```

Comment:
Since `ver` is from `GetLastCommitVersion()`, it is the latest commit version, right? But the commentary says that the current version's value is `ver - 1`, so I confused it. What am I missing?

BTW, how about you write a complete sentence like "Let log stream know the current version".

Reply:
I'm too late. I added comments.